back-end/src/main/kotlin/Gneiss/PacketCompiler/DatabaseAccess/IPacketDao.kt

```diff
@@ -5,4 +5,5 @@ import Gneiss.PacketCompiler.Models.Packet
 interface IPacketDao {
     fun set(key: String, field: String, packet: Packet)
     fun get(key: String, field: String): Packet
+    fun delete(key: String, field: String, packet: Packet): Long
```

Comment:
Why do we need the packet for `delete`?
<img width="909" alt="image" src="https://user-images.githubusercontent.com/62031868/200060812-dd645b73-5231-4019-9436-7d2a2768f446.png">
https://www.javadoc.io/doc/redis.clients/jedis/latest/index.html

Reply:
I see, I'll need a few to review the documentation on that then.